beacon-chain/sync/service.go

```diff
@@ -25,35 +25,47 @@ var log = logrus.WithField("prefix", "sync")
 //     *  Drop peers that send invalid data
 //     *  Throttle incoming requests
 type Service struct {
-	ctx                  context.Context
-	cancel               context.CancelFunc
-	p2p                  types.P2P
-	chainService         types.ChainService
-	announceBlockHashBuf chan p2p.Message
-	blockBuf             chan p2p.Message
+	ctx                         context.Context
+	cancel                      context.CancelFunc
+	p2p                         types.P2P
+	chainService                types.ChainService
+	announceBlockHashBuf        chan p2p.Message
+	blockBuf                    chan p2p.Message
+	announceCrystallizedHashBuf chan p2p.Message
+	crystallizedStateBuf        chan p2p.Message
+	announceActiveHashBuf       chan p2p.Message
+	activeStateBuf              chan p2p.Message
 }
 
 // Config allows the channel's buffer sizes to be changed.
 type Config struct {
-	HashBufferSize  int
-	BlockBufferSize int
+	BlockHashBufferSize             int
+	BlockBufferSize                 int
+	ActiveStateHashBufferSize       int
+	ActiveStateBufferSize           int
+	CrystallizedStateHashBufferSize int
+	CrystallizedStateBufferSize     int
 }
 
 // DefaultConfig provides the default configuration for a sync service.
 func DefaultConfig() Config {
-	return Config{100, 100}
+	return Config{100, 100, 100, 100, 100, 100}
```

Comment:
Oh, I think I wrote this originally but we should make these named struct fields

Reply:
Sure thing. I'll fix that before merge. Thanks for the reminder!